### ISSUE TITLE
Computer Vision Material Suggestion!

### ISSUE THREAD

**Issue by mrdavoodi** [iust-deep-learning/tensorflow-2-tutorial#5]
Here I try to add some contents that I suggest to add for the computer vision section
first of all I think it's good to introduce tensorflow.keras.preprocessing.image.ImageDataGenerator.
we can do lots of preprocessing with this. e.g. rescaling, changing the size of images that have arbitrary sizes to fit the network input shape, shuffling and also data augmentation.
more details can be found here:
https://www.tensorflow.org/tutorials/images/classification

**Comment by kazemnejad** [comment]
Thanks for the suggestion.

Yes! Actually, introducing Data Augmentation methods is a good idea. However, since `tf.data` is the main way for implementing the data pipeline in TensorFlow, I think it would be better to explain data augmentation using `tf.data` and `tf.image`/`tfa.image` ([this](https://www.tensorflow.org/tutorials/images/data_augmentation) can be helpful)

In addition, here are the topics that I think it would be nice to have them:
- Details of all Conv & Pooling layers in TF2 
- TensorFlow Addons goodies for Vision & Image
- Most useful TF hub modules for Vision

@kiamehr74 What is your opinion on this? Do you have any other topics in mind?
